src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2005,6 +2005,52 @@ def delete_ssl_cert(cmd, resource_group_name, certificate_thumbprint):
     raise CLIError("Certificate for thumbprint '{}' not found".format(certificate_thumbprint))
 
 
+def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certificate_name):
+    client = web_client_factory(cmd.cli_ctx)
+    webapp = client.web_apps.get(resource_group_name, name)
+    if not webapp:
+        raise CLIError("'{}' app doesn't exist in resource group {}".format(name, resource_group_name))
+    server_farm_id = webapp.server_farm_id
+    location = webapp.location
+    kv_id = _validate_key_vault_id(cmd.cli_ctx, key_vault, resource_group_name)
+    kv_id_parts = parse_resource_id(kv_id)
+    kv_name = kv_id_parts['name']
+    kv_resource_group_name = kv_id_parts['resource_group']
+    cert_name = '{}-{}-{}'.format(resource_group_name, kv_name, key_vault_certificate_name)
+
+    if not _check_service_principal_permissions(cmd, kv_resource_group_name, kv_name):
+        logger.warning('Unable to verify Key Vault permissions.')
+        logger.warning('You may need to grant Microsoft.Azure.WebSites service principal the Secret:Get permission')
```

Comment:
Add link to the AppService blog here https://azure.github.io/AppService/2016/05/24/Deploying-Azure-Web-App-Certificate-through-Key-Vault.html

Reply:
Added link to warning (and renamed validate function to format per discussion)